### PR TITLE
Add configuration for individual tests from hostonly suite.

### PR DESCRIPTION
**Pull request by godofredoc**
Bug:
  https://github.com/flutter/flutter/issues/63242
  https://github.com/flutter/flutter/issues/64466

### CODE REVIEW

File: config/devicelab_config.star [hunk 1]
```diff
@@ -0,0 +1,437 @@
...
+
+    # Defines triggering policy
+    if branch == "master":
+        triggering_policy = scheduler.greedy_batching(
+            max_concurrent_invocations = 6,
+        )
+    else:
+        triggering_policy = scheduler.greedy_batching(
+            max_batch_size = 1,
+            max_concurrent_invocations = 3,
```
Comment (keyonghan) [inline config/devicelab_config.star:68]: Does this mean `master` have higher priority with `max_concurrent_invocations` set to be bigger?

File: config/devicelab_config.star [hunk 2]
```diff
@@ -0,0 +1,437 @@
+#!/usr/bin/env lucicfg
+# Copyright 2019 The Flutter Authors. All rights reserved.
```
Comment (keyonghan) [inline config/devicelab_config.star:2]: 2020?

File: config/devicelab_config.star (hunk 1 above)
Comment (godofredoc) [reply config/devicelab_config.star:68]: I copied this directly from framework config. I'll adjust it to make it consistent with the engine.

File: config/devicelab_config.star (hunk 2 above)
Comment (godofredoc) [reply config/devicelab_config.star:2]: done

**Review by keyonghan — APPROVED** [review]
LGTM.

**Pull request merged**